CMakeLists.txt

```diff
@@ -172,10 +171,10 @@ set(EXAMPLES_SRC examples/any_return.cpp
                  examples/usertype_var.cpp
                  examples/variables.cpp
                  examples/variadic_args.cpp)
-source_group(EXAMPLES FILES ${EXAMPLES_SRC})
+source_group(examples FILES ${EXAMPLES_SRC})
 foreach(example_source_file ${EXAMPLES_SRC})
-message(STATUS "${example_source_file}")
   get_filename_component(example_name ${example_source_file} NAME_WE)
+  set(example_name "example-${example_name}")
```

Comment:
Unless you use `example_name` again, there's no point using a variable. Just inline this into L178.

Reply:
I use on several lines here. Not sure to understand.

Reply:
My mistake, ignore me. 😨